product_pricelist_fixed_currency_rate/models/product_pricelist_item.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2023 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+
+
+class ProductPricelist(models.Model):
+    _inherit = "product.pricelist.item"
+
+    fixed_currency_rate = fields.Float(
+        digits=(12, 12),
+        help="If set (different to 0.0), the currency conversion will "
+        "ignore the actual currency rate and always use the fixed "
+        "currency rate.",
+    )
+    inverse_fixed_currency_rate = fields.Float(
+        digits=(12, 12),
+        compute="_compute_inverse_fixed_currency_rate",
+        inverse="_inverse_inverse_fixed_currency_rate",
+        help="If set (different to 0.0), the currency conversion will "
+        "ignore the actual currency rate and always use the fixed "
+        "currency rate.",
+    )
+    is_fixed_currency_rate_applicable = fields.Boolean(
+        compute="_compute_is_fixed_currency_rate_applicable"
+    )
+    actual_currency_rate = fields.Float(
+        digits=(12, 12), compute="_compute_is_fixed_currency_rate_applicable"
+    )
+    inverse_actual_currency_rate = fields.Float(
+        digits=(12, 12), compute="_compute_is_fixed_currency_rate_applicable"
+    )
+    do_inverse_currency_rate = fields.Boolean(
+        compute="_compute_do_inverse_currency_rate",
+        store=True,
+        readonly=False,
+    )
+    currency_rate_tooltip = fields.Char(
+        compute="_compute_currency_rate_tooltip",
+    )
+
+    @api.depends("base_pricelist_id", "base_pricelist_id.currency_id")
```

Comment:
to be safe maybe we should also take into account that compute_price must be _formula_. Otherwise, if you have a pricelist that you have configured with fixed rate and then change the logic of the pricelist to fixed price, the fields keep showing in the view:

![image](https://github.com/OCA/product-attribute/assets/71635103/d92ee10f-e829-4c77-ac85-4cab1878ba73)

Reply:
Indeed, attended!